resources/backup-plans.go

```diff
@@ -21,19 +21,30 @@ func init() {
 
 func ListBackupPlans(sess *session.Session) ([]Resource, error) {
 	svc := backup.New(sess)
-	false_value := false
-	max_backups_len := int64(100)
+	falseValue := false
+	maxBackupsLen := int64(100)
 	params := &backup.ListBackupPlansInput{
-		IncludeDeleted: &false_value,
-		MaxResults:     &max_backups_len, // aws default limit on number of backup plans per account
+		IncludeDeleted: &falseValue,
+		MaxResults:     &maxBackupsLen, // aws default limit on number of backup plans per account
 	}
 	resp, err := svc.ListBackupPlans(params)
 	if err != nil {
 		return nil, err
 	}
 
+	backupPlansList := resp.BackupPlansList
+	nextToken := resp.NextToken
+	for nextToken != nil {
```

Comment:
Thank you for your contribution!

In the other resources there is only one loop for fetching the resources, append to the resource slice and handling the next token (https://github.com/rebuy-de/aws-nuke/blob/master/resources/appstream-fleets.go#L17)

I cannot see a reason why not do the same thing here, therefore it would be great if you can change your code to look like the other resources.

Thank you!

Reply:
Thanks for the suggestion, I've modified the change to match the example you provided.